### PR TITLE
Fix SS06 formatting errors

### PR DESCRIPTION
**Pull request by profwacko**
SS06 (Summary should fit in a single line) errors fixed (#29254) for the following:

```
 pandas.DataFrame.slice_shift
 pandas.DataFrame.to_period
 pandas.DataFrame.plot
 pandas.DataFrame.sparse.density
 pandas.DataFrame.style
 pandas.Series.as_blocks
 pandas.Series.ix
 pandas.Series.ptp
 pandas.DataFrame.as_blocks
 pandas.DataFrame.ix
```
Validated with `python scripts/validate_docstrings.py`

Also referencing issue: #27977

### CODE REVIEW

File: pandas/core/arrays/sparse/accessor.py [hunk 1]
```diff
@@ -308,8 +308,10 @@ def to_coo(self):
     @property
     def density(self) -> float:
         """
-        Ratio of non-sparse points to total (dense) data points
-        represented in the DataFrame.
+        Ratio of non-sparse points to total (dense) data points.
+
+        Ratio of non-sparse points to total (dense) data points represented
```
Comment (WillAyd) [inline pandas/core/arrays/sparse/accessor.py]: I think you can remove this extended summary; it says the same thing as the first line

File: pandas/core/generic.py [hunk 2]
```diff
@@ -5766,11 +5766,13 @@ def ftypes(self):
 
     def as_blocks(self, copy=True):
         """
-        Convert the frame to a dict of dtype -> Constructor Types that each has
-        a homogeneous dtype.
+        Convert the frame to a dict of dtype.
```
Comment (WillAyd) [inline pandas/core/generic.py]:
```suggestion
        Convert the frame to a dict of dtype -> Constructor Types.
```

File: pandas/core/generic.py [hunk 3]
```diff
@@ -5766,11 +5766,13 @@ def ftypes(self):
...
 
         .. deprecated:: 0.21.0
 
+        Convert the frame to a dict of dtype-> Constructor Types that each
```
Comment (WillAyd) [inline pandas/core/generic.py]: Remove  this as well; just duplicative

File: pandas/plotting/_core.py [hunk 4]
```diff
@@ -496,6 +496,8 @@ def boxplot_frame_groupby(
 
 class PlotAccessor(PandasObject):
     """
+    Make plots of Series or DataFrame.
+
     Make plots of Series or DataFrame using the backend specified by the
```
Comment (WillAyd) [inline pandas/plotting/_core.py]:
```suggestion
    Uses the backend specified by the
```

File: pandas/core/frame.py [hunk 5]
```diff
@@ -814,8 +814,10 @@ def to_string(
     @property
     def style(self):
         """
-        Property returning a Styler object containing methods for
-        building a styled HTML representation fo the DataFrame.
+        Returns a Styler object.
+
+        Property returning a Styler object containing methods for building
```
Comment (WillAyd) [inline pandas/core/frame.py]: Looks like the word `of` is misspelled here

Comment (WillAyd) [inline pandas/core/frame.py]:
```suggestion
        Contains methods for building a styled HTML representation of the DataFrame.
```

**Review by WillAyd — CHANGES_REQUESTED** [review]
Looks good. Minor comments to improve styling / remove repetition

**Comment by profwacko** [comment]
Hi @WillAyd, added the changes you suggested.

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Great thanks @profwacko !

**Pull request merged**